File: 백준/Bronze/2798. 블랙잭/블랙잭.py
n, m = map(int,input().split())
num = list(map(int,input().split()))
result = []

for i in range(n-2):
    for j in range(i+1,n-1):
        for k in range(j+1,n):
            sum_ = num[i] + num[j] + num[k]
            if sum_ <= m:
                result.append(sum_)               
            else:
                continue
                
print(max(result))

File: 백준/Bronze/2798. 블랙잭/블랙잭.swift
import Foundation

let input = readLine()!.split(separator: " ").map { Int($0)! }
let n = input[0], m = input[1]
let cards = readLine()!.split(separator: " ").map { Int($0)! }
var answer = [Int]()
for i in 0..<cards.count-2 {
    for j in i+1..<cards.count-1 {
        for k in j+1..<cards.count {
            let sumNum = cards[i] + cards[j] + cards[k]
            if sumNum <= m {
                answer.append(sumNum)
            }
        }
    }
}
print(Int(answer.max()!))